tests/tests.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
You haven't written these yet - is this because you have no functions to unit test or because you are planning on doing so later?

When you do come to unit testing, remember to use a separate file to test each function.

Reply:
This is because I have no functions to test at the moment, this will be added to once the functions are written